src/shared/lib/comparison/AnalysisStore.ts

```diff
@@ -0,0 +1,383 @@
+import { remoteData } from 'cbioportal-frontend-commons';
+import {
+    CancerStudy,
+    Gene,
+    Mutation,
+    MutationCountByPosition,
+    Sample,
+} from 'cbioportal-ts-api-client';
+import { computed, makeObservable, observable } from 'mobx';
+import _ from 'lodash';
+import { compileMutations } from '../../../pages/resultsView/ResultsViewPageStoreUtils';
+import internalClient from '../../api/cbioportalInternalClientInstance';
+import { AppStore } from '../../../AppStore';
+import {
+    evaluateMutationPutativeDriverInfo,
+    fetchOncoKbCancerGenes,
+    fetchOncoKbDataForOncoprint,
+    filterAndAnnotateMutations,
+    getGenomeNexusUrl,
+    makeGetOncoKbMutationAnnotationForOncoprint,
+    makeIsHotspotForOncoprint,
+    ONCOKB_DEFAULT,
+} from 'shared/lib/StoreUtils';
+import MobxPromise, { MobxPromiseUnionTypeWithDefault } from 'mobxpromise';
+import { AnnotatedMutation } from '../../../pages/resultsView/ResultsViewPageStore';
+import { DriverAnnotationSettings } from 'shared/alterationFiltering/AnnotationFilteringSettings';
+import { getServerConfig } from 'config/config';
+import { CoverageInformation } from '../GenePanelUtils';
+import { CancerGene, IndicatorQueryResp } from 'oncokb-ts-api-client';
+import {
+    getProteinPositionFromProteinChange,
+    IHotspotIndex,
+    indexHotspotsData,
+    IOncoKbData,
+} from 'cbioportal-utils';
+import { fetchHotspotsData } from '../CancerHotspotsUtils';
+import { GenomeNexusAPIInternal } from 'genome-nexus-ts-api-client';
+import {
+    countMutations,
+    mutationCountByPositionKey,
+} from 'pages/resultsView/mutationCountHelpers';
+import ComplexKeyCounter from '../complexKeyDataStructures/ComplexKeyCounter';
+import GeneCache from 'shared/cache/GeneCache';
+
+export default abstract class AnalysisStore {
```

Comment:
@gblaih is there any reason to worry that these members were changed from what they were in the old resultsviewpagestore?  i assume they were just copied but just want to be sure.

Reply:
the abstract members are different for the resultsviewpagestore vs comparisonstore but the rest are just copied and should have the same behavior.